rpc/backend/backend_suite_test.go

```diff
@@ -0,0 +1,199 @@
+package backend
+
+import (
+	"bufio"
+	"math/big"
+	"os"
+	"path/filepath"
+	"testing"
+
+	dbm "github.com/cometbft/cometbft-db"
+	tmrpctypes "github.com/cometbft/cometbft/rpc/core/types"
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/crypto/keyring"
+	"github.com/cosmos/cosmos-sdk/server"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/ethereum/go-ethereum/common"
+	ethtypes "github.com/ethereum/go-ethereum/core/types"
+	"github.com/evmos/ethermint/app"
+	"github.com/evmos/ethermint/crypto/ethsecp256k1"
+	"github.com/evmos/ethermint/crypto/hd"
+	"github.com/evmos/ethermint/encoding"
+	"github.com/evmos/ethermint/indexer"
+	"github.com/evmos/ethermint/tests"
+	evmtypes "github.com/evmos/ethermint/x/evm/types"
+	"github.com/stretchr/testify/suite"
+
+	"github.com/zeta-chain/zetacore/rpc/backend/mocks"
+	rpctypes "github.com/zeta-chain/zetacore/rpc/types"
+)
+
+type BackendTestSuite struct {
+	suite.Suite
+	backend *Backend
+	acc     sdk.AccAddress
+	signer  keyring.Signer
+}
+
+func TestBackendTestSuite(t *testing.T) {
+	suite.Run(t, new(BackendTestSuite))
+}
+
+const ChainID = "ethermint_9000-1"
```

Comment:
```suggestion
const ChainID = "zetachain_7001-1"
```
ZetaChain localnet chain ID